docs/api-docs/tax/tax-settings.md

```diff
@@ -0,0 +1,92 @@
+# Tax Settings
+
+Tax settings allow you to choose how a store handles tax calculations and price display. For example, you can set how a store handles taxes if an enabled tax service is unavailable. You can also choose whether prices include taxes when displayed on storefront orders. For more info, see [Tax Settings](https://support.bigcommerce.com/s/article/Tax-Overview?language=en_US#tax-settings).
```

Comment:
```suggestion
Tax settings allow you to choose how a store handles tax calculations and price displays. For example, you can set how a store handles taxes if an enabled tax service is unavailable. You can also choose whether prices include taxes when displayed on storefront orders. For more info, see [Tax Settings](https://support.bigcommerce.com/s/article/Tax-Overview?language=en_US#tax-settings).
```

Reply:
Hmm I'm not sure I'd agree with pluralising `display`.